src/IW/App/Error.hs

```diff
@@ -118,7 +123,18 @@ data AppErrorType
     | UrlDownloadFailed Url
     deriving (Show, Eq)
 
--- | Map the @github@ library's @Error@ type into AppErrorType.
+{- | A wrapper around the 'PError' type from the @Cabal@ library.
+This is needed to implement an instance of 'Eq' for 'PError' without
+triggering the @orphan-instances@ warning. -}
+newtype CabalPError = CabalPError 
+    { unCabalPError :: PError 
+    } deriving newtype (Show)
+
+instance Eq CabalPError where
+    (CabalPError (PError pos1 str1)) == (CabalPError (PError pos2 str2)) =
```

Comment:
Outer parenthesis are redundant when defining operator. So we can make code less noisy
```suggestion
    CabalPError (PError pos1 str1) == CabalPError (PError pos2 str2) =
```

Reply:
@chshersh Ok nice. Could this be discovered by hlint?